example/z-playground/client/playground.css

```diff
@@ -39,14 +39,17 @@ body {
 }
 
 
-
+header {
+  height: 64px;
```

Comment:
@aantron Since I'm using browser's save as to adjust the styling, log doesn't show up for me so I'm not sure if this would affected the log and iframe or not. Please check this locally

Reply:
It looks correct!